app/javascript/packages/document-capture/components/documents-step.jsx

```diff
@@ -42,38 +45,60 @@ function DocumentsStep({
   const { notReadySectionEnabled, exitQuestionSectionEnabled, selfieCaptureEnabled } =
     useContext(FeatureFlagContext);
 
-  /**
-   * Sides of document to present as file input.
-   *
-   * @type {DocumentSide[]}
-   */
-  const documentSides = selfieCaptureEnabled ? ['front', 'back', 'selfie'] : ['front', 'back'];
+  const pageHeaderText = selfieCaptureEnabled
+    ? t('doc_auth.headings.document_capture_with_selfie')
+    : t('doc_auth.headings.document_capture');
+
+  const DocumentTipList = withProps({
+    title: t('doc_auth.tips.document_capture_selfie_id_header_text'),
+    titleClassName: 'margin-bottom-0 text-bold',
+    items: [
+      t('doc_auth.tips.document_capture_id_text1'),
+      t('doc_auth.tips.document_capture_id_text2'),
+      t('doc_auth.tips.document_capture_id_text3'),
+    ].concat(!isMobile ? [t('doc_auth.tips.document_capture_id_text4')] : []),
+  })(TipList);
+
+  const defaultSideProps = {
+    registerField,
+    onChange,
+    errors,
+    onError,
+  };
+
+  const DocumentFront = withProps({
+    key: 'front',
+    side: 'front',
+    value: value.front,
+    ...defaultSideProps,
+  })(DocumentSideAcuantCapture);
```

Comment:
Nice! Can you move these outside the `DocumentsStep` component? When they're inline like this it's hard to trace exactly which props are required by each component.

Reply:
This caused some weird test issues on focused element. Revert to the old way.